Add O lettered article stubs from glossary (#3859)
ppy/osu-wiki: wiki/Disambiguation/osu!/en.md

---
stub: true
---

# osu!

**osu!** stylised with a lowercase "o" can refer to three things:

- The game client which hosts all [game modes](/wiki/Game_Modes/) and chat features.
- The game mode unofficially called [osu!standard](/wiki/Game_Modes/osu!/).
- An informal greeting in Japanese, originating by condensing the phrase "Ohayo gozaimasu!" (Good morning!)

>   - Pronounced with a silent u ("OSS").

Reviewer: ```suggestion
  - Pronounced with a silent u ("oh • ss").
```
We should prob also add the IPA spelling of this to reduce ambiguity

  - Close English equivalents include "Yo!" or "Hey!".

> *For more information about its meaning, see [Does anyone here speak/learn Japanese?](https://osu.ppy.sh/community/forums/posts/2686).*

Reviewer: ```suggestion
*For more information about osu! as an informal greeting, see: [Does anyone here speak/learn Japanese?](https://osu.ppy.sh/community/forums/posts/2686).*
```
